flask/requirements.txt

```diff
@@ -1,3 +0,0 @@
-Flask==1.0.3
```

Comment:
This requirements file shouldn't have been deleted

Reply:
Sorry, not sure how that happened.